### PR TITLE
Feature/i18n

### PR DESCRIPTION
**Pull request by MykhailoKutsyi**
Hello 
___
This your message 

lib does not have exmple, move [example-button] to lib 2)connect absolute path!!!, 3) all text should be in file text, use i18n approach (just added to instruction)
___

1. Fixed:  created new example-button as lib.
2. Fixed absolute path
3.  Added i18n. For now, all text in translations/index.ts 

Attached screenshot of how it looks. 
image

### CODE REVIEW

File: apps/freelance/src/pages/ExampleRootPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import { Link } from 'react-router-dom';
+import { ExampleButton } from '@freelance/components';
+import { Wrapper, Title, Text } from './styles';
+
+import { useGetPokemonByNameQuery } from '../../redux/services/pokemon';
```
Comment (lyudad) [inline apps/freelance/src/pages/ExampleRootPage/index.tsx:7]: absolute path

File: apps/freelance/src/pages/ExampleRootPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+export default function ExampleRootPage() {
+  const { t } = useTranslation();
+
+  const { data, error, isLoading } = useGetPokemonByNameQuery('bulbasaur');
+
+  return (
+    <Wrapper>
+      <Title>{t('description.root page')}</Title>
+      <h2>{t('description.Welcome to React')}</h2>
+      <Text>{t('description.reusable component')}</Text>
```
Comment (lyudad) [inline apps/freelance/src/pages/ExampleRootPage/index.tsx:19]: no spaces, never never never in constants
bad - description.reusable component
good - description.reusable_component, description.reusableComponent

File: apps/freelance/src/translations/index.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  .init({
+    debug: true,
+    fallbackLng: 'en',
+    interpolation: {
+      escapeValue: false, // not needed for react as it escapes by default
+    },
+    resources: {
+      en: {
+        translation: {
+          description: {
```
Comment (lyudad) [inline apps/freelance/src/translations/index.tsx:22]: no, it should be separate file

File: libs/components/src/lib/components.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+export interface ComponentsProps {}
+
+const StyledComponents = styled.div`
+  color: pink;
+`;
+
+export function Components(props: ComponentsProps) {
+  return (
+    <StyledComponents>
+      <h1>Welcome to Components!</h1>
```
Comment (lyudad) [inline libs/components/src/lib/components.tsx:13]: or delete text or i18n, no text should be in components

Comment (MykhailoKutsyi) [reply libs/components/src/lib/components.tsx:13]: deleted

File: apps/freelance/src/pages/ExampleRootPage/index.tsx (hunk 2 above)
Comment (MykhailoKutsyi) [reply apps/freelance/src/pages/ExampleRootPage/index.tsx:19]: updated

**Pull request merged**